democracy_club/templates/base.html

```diff
@@ -5,128 +5,65 @@
-    <link rel="manifest" href="{% static "manifest.json" %}">
-    {% block site_icons %}
-        <link rel="shortcut icon" href="{{ CANONICAL_URL }}{% static 'images/logo_icon.png' %}">
-        <link rel="apple-touch-icon" sizes="180x180" href="{{ CANONICAL_URL }}{% static 'icons/apple-touch-icon.png' %}">
-        <link rel="icon" type="image/png" href="{{ CANONICAL_URL }}{% static 'icons/favicon-32x32.png' %}" sizes="32x32">
-        <link rel="icon" type="image/png" href="{{ CANONICAL_URL }}{% static 'icons/favicon-16x16.png' %}" sizes="16x16">
-        <link rel="mask-icon" href="{{ CANONICAL_URL }}{% static 'icons/safari-pinned-tab.svg' %}" color="#ec008c">
-    {% endblock site_icons %}
-    {% block base_og_tags %}
-        <meta name="description" property="og:description" content="{% block og_description %}We build digital tools to support everyone’s participation in UK democracy.{% endblock og_description %}" />
-        <meta name="image" property="og:image" content="{% block og_image_content %}{{ CANONICAL_URL }}{% static 'images/og_image_logo.png' %}{% endblock og_image_content %}" />
-        <meta name="title" property="og:title" content="{% block og_title_content %}{{ SITE_TITLE }}{% endblock og_title_content %}" />
-        <meta name="type" property="og:type" content="{% block og_type_content %}website{% endblock og_type_content %}">
-        <meta name="url" property="og:url" content="{% block og_url_content %}{{ CANONICAL_URL }}{{ request.path }}{% endblock og_url_content %}">
-    {% endblock base_og_tags %}
-    {% block twitter_tags %}
-        <meta name="twitter:card" content="{% block twitter_card_type %}summary{% endblock twitter_card_type %}">
-        <meta name="twitter:site" content="@democlub">
-        <meta name="twitter:image" content="{% block twitter_image %}{{ CANONICAL_URL }}{% static 'images/logo_icon.png' %}{% endblock twitter_image %}" />
-        <meta name="twitter:image:alt" content="{% block twitter_image_alt %}Democracy Club logo{% endblock twitter_image_alt %}">
-        <meta name="twitter:title" content="{% block twitter_title_content %}Democracy Club{% endblock twitter_title_content %}">
-        <meta name="twitter:url" content="{{ CANONICAL_URL }}{{ request.path }}">
-        <meta name="twitter:description" content="{% block twitter_description_content %}We build digital tools to support everyone’s participation in UK democracy.{% endblock twitter_description_content %}"/>
-    {% endblock twitter_tags %}
-{% endblock site_meta %}
-
 {% block extra_site_css %}
     {% stylesheet 'styles' %}
 {% endblock extra_site_css %}
 
-{% block body_base %}
-    <body class="ds-width-full">
-        <div class="ds-page">
-            <p><a class="ds-skip-link" href="#main">skip to content</a></p>
-            {% block header %}
-                {% block site_menu %}
-                    <header class="ds-header">
-                        <a class="ds-logo" href="/">
```

Comment:
Currently, we have `SITE_WIDTH_LOGO` set in `dc_utils` context processors to `"300"`. I've tried to override this locally in `base.py` to fix the logo spacing issue shown here, but it remains at 300:

<img width="655" alt="Screen Shot 2022-10-21 at 5 07 23 PM" src="https://user-images.githubusercontent.com/7017118/197241460-9b026759-2e4f-49af-98d8-7528d654b703.png">

Reply:
Ignore this; it turns out this is unrelated to settings. This spacing is on the `img` tag.